### DISCUSSION TITLE
Inquiries regarding Nextcloud updates

### DISCUSSION THREAD

**Discussion by NousernameSad2** [runtipi/runtipi-appstore#5784] [Q&A]
Hi there! I'm fairly new to Runtipi and I don't have much background information regarding how your github repositories work. Is there a place where we can request for updates? 

Currently, I'm running Nextcloud on my runtipi server but noticed that it is still at version 26.0.13 (already end of life). Is there any way to get to the latest version without breaking anything? Version 26 is multiple generations behind, and most plugins do not support this anymore.

**Comment by OriginalSimon** [comment]
![зображення](https://github.com/user-attachments/assets/55b1ae12-5570-4d9b-9cec-a8a1eacb5149)
Only this option, it includes updates. Join the Discord and find out why.

**Reply by NousernameSad2** [reply]
Thanks for the swift reply! Will check it out :D

Will also close the discussion now.
